src/components/ui/accessibility/VisuallyHidden.tsx

```
@@ -0,0 +1,25 @@
+import { ReactNode } from "react";
+
+export default function VisuallyHidden({
+  children: children,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)